deepchecks/utils/single_sample_metrics.py

```diff
@@ -24,26 +24,30 @@ def calculate_neg_mse_per_sample(labels, predictions, index=None) -> pd.Series:
     return pd.Series([-(y - y_pred) ** 2 for y, y_pred in zip(labels, predictions)], index=index)
 
 
-def calculate_neg_cross_entropy_per_sample(labels, probas: np.ndarray, model_classes: Optional[List] = None,
-                                           index=None, eps=1e-15) -> pd.Series:
+def calculate_neg_cross_entropy_per_sample(labels, probas: np.ndarray,
```

Comment:
add tests for multi-label (can copy structure from my PR)
Most important to verify running it on the multilabel format for regular classification return the same result